stream_alert_cli/terraform/classifier.py

```diff
@@ -69,8 +69,8 @@ def generate_classifier(cluster_name, cluster_dict, config):
         'function_role_id': '${{module.{}_lambda.role_id}}'.format(tf_module_prefix),
         'function_alias_arn': '${{module.{}_lambda.function_alias_arn}}'.format(tf_module_prefix),
         'function_name': '${{module.{}_lambda.function_name}}'.format(tf_module_prefix),
-        'classifier_sqs_queue_arn': '${module.globals.classifier_sqs_queue_arn}',
-        'classifier_sqs_queue_url': '${module.globals.classifier_sqs_queue_url}',
```

Comment:
The classifier terraform module was not using the URL. It was used in `SQS_QUEUE_URL` environment variable, which is below.

Reply:
thanks for the note